weasyl/search.py

```diff
@@ -31,6 +31,8 @@
     "journal": (30, "j", "journal", "journalid", 3999),
 }
 
+count_limit = 10000
```

Comment:
Although this is tweaked in tests, it is treated as a constant everywhere else. Should it be COUNT_LIMIT?